src/legacy/core_plugins/vis_default_editor/public/_default.scss

```
@@ -72,6 +72,7 @@
   display: flex;
   flex-basis: 100%;
   flex: 1;
+  overflow: auto;
```

Ok I've tested it further and can reproduce. I had to shrink my screen's width in order to see the sidebar width get smaller than the min-width of `350px`. 

Can you change this to `hidden` though, as then we know we won't get unwanted scrollbars and it will force the inner child to show the scrollbars instead.
```suggestion
  overflow: hidden;
```

---

Ok, done